docs/design/02-pointers.md

```diff
@@ -73,6 +73,17 @@ Like this?
 }
 ```
 
-As you can see, none of these interpretations are accurate. When turning into JSON, Ardite will prefer the second format if the `/profile/about` format is the most recent (recency will be better explained when we talk about the ledger). However, if `/profile` is the most recent, Ardite will prefer the first JSON format. Ardite would never create a document which looks like the third format (for more information on how this is interpreted, see the reducing section of the design docs).
+As you can see, none of these interpretations are very helpful. When turning into JSON the third format is what would naturally be assumed, where the second value will be whatever is the most recent (recency will be better explained when we talk about the ledger). This is a very ugly solution, therefore in most scenarios Ardite prefers the following format:
```

Comment:
Change to:
As you can see, the first and second one hide parts of the state whilst the third is not valid JSON. Thus Ardite will provide the following format unless a schema is applied: